mods/common/languages/en.ftl

```diff
@@ -117,469 +119,468 @@ overwrite-map-outside-edit-prompt = "The map has been edited from outside the ed
 
 ## InstallFromDiscLogic
-detecting-drives = Detecting drives
-checking-discs = Checking Discs
-searching-disc-for = Searching for { $title }
-content-package-installation = The following content packages will be installed:
-game-discs = Game Discs
-digital-installs = Digital Installs
-game-content-not-found = Game Content Not Found
-alternative-content-sources = Please insert or install one of the following content sources:
-installing-content = Installing Content
-copying-filename = Copying { $filename }
-copying-filename-progress = Copying { $filename } ({ $progress }%)
-installation-failed = Installation Failed
-check-install-log = Refer to install.log in the logs directory for details.
-extracting-filename = Extracting { $filename }
-extracting-filename-progress = Extracting { $filename } ({ $progress }%)
-cancel = Cancel
-retry = Retry
+label-detecting-drives = Detecting drives
+label-checking-discs = Checking Discs
+label-searching-disc-for = Searching for { $title }
+label-content-package-installation = The following content packages will be installed:
+label-game-discs = Game Discs
+label-digital-installs = Digital Installs
+label-game-content-not-found = Game Content Not Found
+label-alternative-content-sources = Please insert or install one of the following content sources:
+label-installing-content = Installing Content
+label-copying-filename = Copying { $filename }
+label-copying-filename-progress = Copying { $filename } ({ $progress }%)
+label-installation-failed = Installation Failed
+label-check-install-log = Refer to install.log in the logs directory for details.
+label-extracting-filename = Extracting { $filename }
+label-extracting-filename-progress = Extracting { $filename } ({ $progress }%)
+button-cancel = Cancel
+button-retry = Retry
 
 ## InstallFromDiscLogic, LobbyLogic
-back = Back
+button-back = Back
 
 # InstallFromDiscLogic, ModContentPromptLogic
-continue = Continue
+button-continue = Continue
 
 ## ModContentLogic
-manual-install = Manual Install
+button-manual-install = Manual Install
 
 ## ModContentPromptLogic
-quit = Quit
+button-quit = Quit
 
 ## KickClientLogic
-kick-client = Kick { $player }?
+dialog-prompt-kick-client = Kick { $player }?
 
 ## KickSpectatorsLogic
-kick-spectators =
+dialog-prompt-kick-spectators =
     { $count ->
         [one] Are you sure you want to kick one spectator?
        *[other] Are you sure you want to kick { $count } spectators?
     }
 
 ## LobbyLogic
-add = Add
-remove = Remove
-configure-bots = Configure Bots
-n-teams = { $count } Teams
-humans-vs-bots = Humans vs Bots
-free-for-all = Free for all
-configure-teams = Configure Teams
-
-## LobbyLogic, CommonSelectorLogic, InGameChatLogic
-all = All
+label-add-bots = Add
+label-remove-bots = Remove
+label-configure-bots = Configure Bots
+label-teams-count = { $count } Teams
+label-humans-vs-bots = Humans vs Bots
+label-free-for-all = Free for all
+label-configure-teams = Configure Teams
```

Comment:
these are buttons

Reply:
They appear as options in a dropdown that act like buttons... That's why I opted for the more generic "label" prefix here. But you have a point - essentially they are buttons, just dressed in a weird way.